superset-frontend/src/cccs-viz/plugins/plugin-chart-cccs-grid/src/plugin/controlPanel.ts

```diff
@@ -378,28 +440,6 @@ const config: ControlPanelConfig = {
 // CLDN-941: Only show the CUSTOMIZE tab if DASHBOARD_CROSS_FILTERS are enabled in the system.
```

Comment:
Maybe you could remove this comment or move it above where this code was moved to. Otherwise it will just be sitting by itself